### PR TITLE
Update volunteer application 

### PR DESCRIPTION
**Pull request by anivorlis**
The form is closed, we are no longer accepting applicants through the form. We have made the selection and are awaiting for responses. If people want to volunteer some of their time, it would be greatly appreciated but as of now we cannot offer them a free ticket.

### CODE REVIEW

**Comment by cybit** [comment]
This seems to be based on an old version of main. (sessions.mdx, tickets.mdx)
Could you please do a rebase of your PR?

**Comment by anivorlis** [comment]
Correct, @cybit please review now :)

File: src/content/pages/volunteers.mdx [hunk 1]
```diff
@@ -95,13 +95,13 @@ There are always some tasks which we simply cannot plan beforehand. Based on wha
...
 
-Please acknowledge our [Data Privacy Policy](https://forms.gle/PCP7bXvdH1QhDBPn6) first, and then fill out the form below to register your interest in volunteering at EuroPython 2024!
-We will get back to you with more details and information on how to proceed.
-Thanks ❤️
+Quite a lot of enthusiastic Pythonistas expressed their interest to volunteer and with so many great applications,
+making a decision was tough this year. Unfortunately, at this point we are not able to offer you a ticket for your volunteer contribution.
+We have a limited number of tickets for volunteers and we are unable to accommodate everyone.
+
+If you can make it to the conference and you’d like to volunteer some of your time we’d be truly grateful. Even though we cannot offer you a ticket,
+we will gladly welcome you to our volunteer family! ❤️ If you are interest, you can mail us at [volunteers@europython.eu](mailto:volunteers@europython.eu)
```
Comment (hypha) [inline src/content/pages/volunteers.mdx:103]: modified it a bit to hopefully make it a bit natural and explicit. 
```suggestion
Thank you to all the enthusiastic Pythonistas who expressed interest in volunteering! With so many great applications, making a decision was tough this year. Unfortunately, we have already allocated all our volunteer tickets and are no longer able to offer you one at this time. Due to our limited capacity, we cannot accommodate everyone.

However, if you're able to attend the conference and still wish to volunteer, we would truly appreciate your support. While we cannot provide a ticket, we warmly welcome you to join our volunteer family! ❤️ If you're interested, please email us at [volunteers@europython.eu](mailto:volunteers@europython.eu).
```

Comment (anivorlis) [reply src/content/pages/volunteers.mdx:103]: Thank you!

**Comment by cybit** [comment]
Not sure how to add suggestions to these lines:
Line 19:
```
* You are welcome to volunteer as much or as little as you want to, you just need to hold a ticket. We appreciate any help you can give us.
```
Line 23:
remove

**Comment by anivorlis** [comment]
Thank you for the comment @cybit, I updated the language after your comment (hopefully I understood correctly).
If that is not the case let me know

**Comment by cybit** [comment]
My issue is with lines 19 and 23, @anivorlis . 
https://github.com/EuroPython/website/blob/8b4856936a65917aa62c82e14b30d444d956df0d/src/content/pages/volunteers.mdx?plain=1#L19
https://github.com/EuroPython/website/blob/8b4856936a65917aa62c82e14b30d444d956df0d/src/content/pages/volunteers.mdx?plain=1#L23

You need a ticket (19)
And getting one is no longer possible so why have it as a perk ?(23)

**Comment by anivorlis** [comment]
Oh sorry, I thought you meant to rephrase the second paragraph. My bad, i will revert it. 

Line 19: it is still possible
Line 23: I think it is a perk that we gave to some volunteers - as many as we could. imo is better to keep it for transparency but I am fine with removing it

wdyt @cybit ?

**Comment by cybit** [comment]
Line 19: is is possible but you need a ticket. That's why up suggested.
```
* You are welcome to volunteer as much or as little as you want to, you just need to hold a ticket. We appreciate any help you can give us.
``` 
If you prefer the "even if you hold a ticket" phrase. Not the important stuff.

Line23: I would remove it. Old people will get their voucher , new people will not fall over it. And with line 99 there should be an indication that yes, we did offer tickets, but they are out.

**Comment by anivorlis** [comment]
Agree on your suggestion for line 19: changed already hold with need to hold
Line 23: maybe use strikethrough to indicate it was an option but no longer available? Alternatively we can used [CLOSED] inline with finaid

I like the website to reflect that this was a perk, which we gave to as many people as we could.

**Review by cybit — APPROVED** [review]
Looks good now. Thank you.

File: src/content/pages/volunteers.mdx [hunk 2]
```diff
@@ -95,13 +95,11 @@ There are always some tasks which we simply cannot plan beforehand. Based on wha
 
 # How to sign up
 
-Please acknowledge our [Data Privacy Policy](https://forms.gle/PCP7bXvdH1QhDBPn6) first, and then fill out the form below to register your interest in volunteering at EuroPython 2024!
-We will get back to you with more details and information on how to proceed.
-Thanks ❤️
+Thank you to all the enthusiastic Pythonistas who expressed interest in volunteering! With so many great applications, making a decision was tough this year.
+Unfortunately, we have already allocated all our volunteer tickets and are no longer able to offer you one at this time. Due to our limited capacity, we cannot accommodate everyone.
```
Comment (laisbsc) [inline src/content/pages/volunteers.mdx:99]: I think we should swap the order of these two paragraphs. Make it obvious we're full first and then thank people.

**Review by laisbsc — APPROVED** [review]
A tiny comment. Looks good, otherwise. Good to merge 🥳

File: src/content/pages/volunteers.mdx (hunk 2 above)
Comment (anivorlis) [reply src/content/pages/volunteers.mdx:99]: It was merged but we can still change it if you think it is needed. 
Maybe read it once on the website and let us know?

**Pull request merged**